src/index.js

```diff
@@ -25,10 +25,62 @@ import cors from 'cors';
 import ip from 'ip';
 import ProxyAgent from 'proxy-agent';
 
+const moduleNames = [];
+const rootDirectory = process.env.HOMEPATH || process.env.HOME;
```

Comment:
This is a huge nit pick, but `root directory` already has a very strong meaning in unix and linux operating systems.

For that sake, can this be called 'cacheDirectory' please :)